test/e2e/apimachinery/discovery.go

```diff
@@ -159,7 +159,14 @@ var _ = SIGDescribe("Discovery", func() {
 		}
 	})
 
-	ginkgo.It("should locate the groupVersion and a resource within each APIGroup", func(ctx context.Context) {
+	/*
+		Release: v1.28
+		Testname: Discovery, confirm the groupVerion and a resourcefrom each apiGroup
+		Description: A resourceList MUST be found for each apiGroup that is retrieved.
+		For each apiGroup the groupVersion MUST equal the groupVersion as reported by
+		the schema. From each resourceList a valid resource MUST be found.
```

Comment:
Is the list of items in `test` an exhaustive list? Would we have to keep it updated every release?

Reply:
AFAIK, the list of items in `test` covers the current list of GA apiGroups at the moment. The test would only need to be updated when a new apiGroup goes GA.

Reply:
cc @liggitt ok with you? do we need to find a way to raise the flag automatically?

Reply:
this is better than what we have today... automatically catching new groups would be nice but I wouldn't block here on it

Reply:
One of ii's action items is to create the release informing job that watch for new endpoints graduating to GA.
In the mean time we are doing a weekly update of APISnoop and watch for any changes in GA endpoints.

Reply:
We raise the flag. 
Manually at the moment. :south_africa: :smile: